data_owner_ids.py

```diff
@@ -18,30 +24,99 @@ def parse_args():
     return args
 
 
-def process_csv(csv_path, system_csv_path, system):
-    with open(csv_path) as csvfile:
+def zip_and_clean(system_output_path, system, timestamp):
+    with zipfile.ZipFile(
+        system_output_path.parent / f"{system}.zip", mode="w"
+    ) as system_archive:
+        system_archive.write(
+            system_output_path / f"{system}{timestamp}.csv",
```

Comment:
We should continue to use the `subject-timestamp.csv` convention we established in data-owner-tools (See this discussion: https://github.com/mitre/data-owner-tools/pull/29#discussion_r961697265)

Can you add a `-` before the timestamp in the necessary places?